src/SharedUtilities/models.jl

```diff
@@ -226,7 +226,7 @@ have different dimensions - require defining a new method.
 """
 function initialize_prognostic(
     model::AbstractModel{FT},
-    state::Union{ClimaCore.Fields.Field, Vector{FT}},
+    state::Union{ClimaCore.Fields.Field, ClimaCore.Fields.FieldVector, Vector},
```

Comment:
This change will result in the `detect_unbound_args_recursively` test to fail-- `FT` is removed from the argument type parameters, but still exists in the `where {FT}` part

Reply:
Oops, I didn't mean to remove the `{FT}`. I'll add it back.

Reply:
Wait, just to make sure I understand `detect_unbound_args_recursively`, would we expect it to fail in this case even though we have `AbstractModel{FT}` with the `where {FT}`? It looks like all the aqua tests pass for this commit

Reply:
Oh, whoops! I missed the `AbstractModel{FT}`, so `FT` is always present (and the tests should pass)